### PR TITLE
Added Flickr images to UI

### PR DESCRIPTION
**Pull request by DecDury**
* Implemented Jadrien's scraped images into the main feed
* made this branch up to date with the changes in dev
* some minor refactoring

### CODE REVIEW

File: app/src/main/java/com/example/jwstimager/IconData.kt [hunk 1]
```diff
@@ -4,33 +4,33 @@ object IconData {
     val IconDataList = listOf<ImageData>(
```
Comment (DecDury) [inline app/src/main/java/com/example/jwstimager/IconData.kt:4]: What was IconData.kt used for? I had to comment some stuff out when I changed some of the attributes of the ImageData class. Do we still need this file?

**Comment by DecDury** [comment]
Also are you guys good if we remove the SampleFavorites and initial SampleData classes I added. No point in having them in the final product.

File: app/src/main/java/com/example/jwstimager/IconData.kt (hunk 1 above)
Comment (Drew-Woodz) [reply app/src/main/java/com/example/jwstimager/IconData.kt]: IconData.kt is a depreciated data structure for nav. It should have no dependencies and can be deleted.

**Review by Drew-Woodz — APPROVED** [review]
Looks Good!

**Review by jadrienhannon — APPROVED** [review]

**Pull request merged**